incubator/java-microprofile/image/project/Dockerfile

```diff
@@ -1,23 +1,34 @@
+# Step 1: Build the user's application
 FROM adoptopenjdk/openjdk8-openj9
 
 RUN apt-get update && \
     apt-get install -y maven unzip wget xmlstarlet
 
-COPY . /project
+# Copy and build the dev.appsody:java-microprofile parent pom
+COPY ./pom.xml /project/pom.xml
+RUN cd /project && mvn -B install dependency:go-offline -DskipTests
 
+# Copy and run a simple version check
+COPY ./util /project/util
 RUN  /project/util/check_version build
```

Comment:
This isn't running the check, it's just building the check_version utility, that's invoked via the validate.sh, except, validate.sh has fallen off all paths except init, where it's a tad pointless.